WeightProducer/python/MCSamples_Summer20UL16APV.py

```diff
@@ -90,6 +90,7 @@
     MCSample('WJetsToQQ_HT-800toInf_TuneCP5_13TeV-madgraphMLM-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v2', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 7740501),
     MCSample('QCD_HT1000to1500_TuneCP5_PSWeights_13TeV-madgraphMLM-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v1', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 13823774),
     MCSample('QCD_HT100to200_TuneCP5_PSWeights_13TeV-madgraphMLM-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v1', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 67431856),
+    MCSample('QCD_HT200to300_TuneCP5_PSWeights_13TeV-madgraph-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v1', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 18273591),
```

Comment:
HT300to500 seems to be available now as well. You might as well add that sample in this PR.

Reply:
Hi @aperloff ,
Sure I can get that in. Besides these fixes, I can merge in my other branch here, which takes care of samples missing from dictionaries/new samples. I was not sure if we wanted to keep this branch just for quick fixes.

Reply:
Ummm...maybe just keep the PRs separate. It's easier to peel back the history is changes are small and well contained.

Reply:
Yep, totally fine with me. I did just add in the HT300to500 sample.